### PR TITLE
Rebase from PR #50

### PR DESCRIPTION
**Pull request by oupadhyay**
Rebase from PR #50

### CODE REVIEW

File: packages/client/src/models/mentor.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { Db } from '../providers/FirebaseProvider';
+import * as yup from 'yup';
```
Comment (sanjayyepuri) [inline packages/client/src/models/mentor.js:2]: can you make `yup` start with a capital "Y." This is just a nit.

File: packages/client/src/models/mentor.js [hunk 2]
```diff
@@ -20,39 +29,110 @@ const MentorConverter = {
...
             data.name,
             data.email,
+            data.phone,
+            data.pronouns,
+            data.college,
+            data.avatar,
+            data.bio,
+            data.major,
             data.timezone,
+            // what is this for?
```
Comment (sanjayyepuri) [inline packages/client/src/models/mentor.js:39]: Does the schema have both "bio" and "about"?

**Comment by oupadhyay** [comment]
Closes #50

File: packages/client/src/models/mentor.js (hunk 2 above)
Comment (oupadhyay) [reply packages/client/src/models/mentor.js:39]: yeah

**Pull request merged**